framework/src/modules/chain/synchronizer/utils.js

```diff
@@ -42,6 +42,29 @@ const restoreBlocks = async (blocksModule, processorModule, tx) => {
 	return true;
 };
 
+/**
+ * Deletes blocks of the current chain after the desired height exclusive and
+ * backs them up in temp_block database table.
+ * @param {Object} processorModule
+ * @param {Object} blocksModule
+ * @param {Number} desiredHeight - The height desired to delete blocks after.
+ * @return {Promise<void>} - Promise is resolved when blocks are successfully deleted
+ */
+const deleteBlocksAfterHeightAndBackup = async (
```

Comment:
I think I added a comment about the naming of this function? `AndBackup` maybe `saveToTemp` to keep consistency or is that not related? @limiaspasdaniel

Reply:
I think `backup` is a synonym to `saveToTemp` and it better wraps the meaning of backing blocks up.